packages/server/lib/util/editors.ts

```diff
@@ -2,64 +2,40 @@ import _ from 'lodash'
 import Bluebird from 'bluebird'
 import debugModule from 'debug'
 
-import { getEnvEditors, Editor } from './env-editors'
+import { Editor, osFileSystemExplorer, EditorsResult } from '@packages/types'
+import { getEnvEditors } from './env-editors'
 import shell from './shell'
 import savedState from '../saved_state'
 
 const debug = debugModule('cypress:server:editors')
 
-interface CyEditor {
-  id: string
-  name: string
-  openerId: string
-  isOther: boolean
-}
-
-interface EditorsResult {
-  preferredOpener?: CyEditor
-  availableEditors?: CyEditor[]
-}
-
-const createEditor = (editor: Editor): CyEditor => {
+const createEditor = (editor: Editor): Editor => {
   return {
     id: editor.id,
     name: editor.name,
-    openerId: editor.binary,
-    isOther: false,
+    binary: editor.binary,
   }
 }
 
-const getOtherEditor = (preferredOpener?: CyEditor) => {
+const getOtherEditor = (preferredOpener?: Editor): Editor | undefined => {
   // if preferred editor is the 'other' option, use it since it has the
-  // path (openerId) saved with it
-  if (preferredOpener && preferredOpener.isOther) {
+  // path (binary) saved with it
+  if (preferredOpener && preferredOpener.id === 'other') {
     return preferredOpener
   }
 
-  return {
-    id: 'other',
-    name: 'Other',
-    openerId: '',
-    isOther: true,
```

Comment:
Removed this option from here - it's only here for the purpose of having a placeholder on the front-end UI.

We still need that placeholder for now, so I append it here... https://github.com/cypress-io/cypress/pull/18932/files#diff-164ce6bedd1fe744fbd75f7b8526d850c94eb6fd43b9c22e60b18e976d9b21a8R376 will delete post 10.0.